components/combobox/private/menu.jsx

```diff
@@ -156,99 +160,160 @@ const Menu = (props) => {
 			);
 		}
 
-		return (
-			<li
-				className="slds-listbox__item"
-				key={`menu-option-${optionData.id}`}
-				role="presentation"
-			>
-				{
-					{
-						'icon-title-subtitle': (
-							<span // eslint-disable-line jsx-a11y/no-static-element-interactions
-								aria-selected={active}
-								id={`${props.inputId}-listbox-option-${optionData.id}`}
+		const disabledProps = {};
+		const tooltipId = `${props.inputId}-listbox-option-help-${optionData.id}`;
+		if (optionData.disabled) {
```

Comment:
These should only be applied to the node if the tooltip is present and only when the tooltip is showing--otherwise it's an invalid DOM id for assistive tech.
![screen shot 2019-02-05 at 11 40 49 am](https://user-images.githubusercontent.com/1290832/52296217-08dcad00-293b-11e9-9bc2-2c6e09fa7417.png)

Reply:
oh makes sense - but I think only for`disabledProps['aria-describedby'] = tooltipId`. 

I still need 
`disabledProps['aria-disabled']` and `disabledProps.style = {cursor: 'default'}` 
in cases where a consumer doesn't use a tooltip.

Reply:
Yes. You are right.